### PR TITLE
CI runs Clippy

### PR DESCRIPTION
**Pull request by 1tgr**

### CODE REVIEW

File: src/derived/slot.rs [hunk 1]
```diff
@@ -663,6 +664,7 @@ where
     /// that work completes. This helper does that; it returns a port
     /// where you can wait for the final value that wound up being
     /// computed (but first drop the lock on the map).
+    #[allow(clippy::type_complexity)]
```
Comment (nikomatsakis) [inline src/derived/slot.rs:667]: Can we just put this allow at the crate level? I'm not into this lint.

File: tests/parallel/independent.rs [hunk 2]
```diff
@@ -8,11 +8,11 @@ fn in_par_two_independent_queries() {
     let mut db = ParDatabaseImpl::default();
 
     db.set_input('a', 100);
-    db.set_input('b', 010);
-    db.set_input('c', 001);
+    db.set_input('b', 10);
+    db.set_input('c', 1);
     db.set_input('d', 200);
-    db.set_input('e', 020);
-    db.set_input('f', 002);
```
Comment (nikomatsakis) [inline tests/parallel/independent.rs:15]: the formatting here was intentional, but...it's fine

**Review by nikomatsakis — CHANGES_REQUESTED** [review]
I've never really used clippy before, but for the most part these changes seem good.

File: tests/parallel/independent.rs (hunk 2 above)
Comment (1tgr) [reply tests/parallel/independent.rs:15]: I agree it's not a big deal, but maybe 200, 220, 202?

**Comment by nikomatsakis** [comment]
cargo fmt errors

**Review by nikomatsakis — APPROVED** [review]

**Comment by nikomatsakis** [comment]
bors r

**Pull request merged**